CHANGELOG.md

```diff
@@ -16,6 +17,11 @@ All notable changes to this project will be documented in this file. The format
 
 ## [Unreleased]
 
+## [1.1.103] - 2024-08-13
+
+### Added
+- [Double Spend Detection](https://bitcoin-sv.github.io/arc/#/?id=double-spending) is a feature that introduces `DOUBLE_SPEND_ATTEMPTED` status to transactions that attempt double spend together with `CompetingTxs` field in the API responses and callbacks.
```

Comment:
@arkadiuszos4chain Let's add here also the cumulative fee validation feature

Reply:
Done